app/src/main/java/org/oppia/app/home/recentlyplayed/RecentlyPlayedFragmentPresenter.kt

```diff
@@ -98,7 +98,7 @@ class RecentlyPlayedFragmentPresenter @Inject constructor(
         )
         else -> {
           logger.d("RecentlyPlayedFragment", "Successfully loaded exploration")
-          routeToExplorationListener.routeToExploration(internalProfileId, topicId, storyId, explorationId)
+          routeToExplorationListener.routeToExploration(internalProfileId, topicId, storyId, explorationId,null)
```

Comment:
Nit: whenever we pass null or an integer or a boolean value we write inline comment to make it more readable code.
`routeToExplorationListener.routeToExploration(internalProfileId, topicId, storyId, explorationId, /* backflowId= */ null)`

Reply:
Okay, done :)